modules/win_mozilla_build/manifests/tooltool.pp

```diff
@@ -29,4 +32,16 @@
                                         },
         inherit_parent_permissions => true,
     }
+    file { "${builds}\\relengapi.tok":
+        content   => $win_mozilla_build::tooltool_tok,
+        show_diff => false,
+    }
+    # This script will download and add the certifictae form https://tooltool.mozilla-releng.net
+    # Without tooltool will fail during tasks
```

Comment:
I'm not sure what the comments mean here.

Reply:
That was very poorly worded. It is now updated.